src/pqm/pqm.py

```diff
@@ -185,10 +534,10 @@ def pqm_pvalue(
         x_samples, y_samples, and/or a Gaussian distribution, see the note
         below.
     re_tessellation : Optional[int]
-        Number of times pqm_pvalue is called, re-tesselating the space. No
+        Number of times _pqm_test is called, re-tesselating the space. No
```

Comment:
I don't think the user needs to know that we've organized the code to use a common `_pqm_test` function, just saying that the code is rerun a bunch of times is enough